src/modules/AnalysisExecution/components/ViewEditAnalysis/ModalFiles/presenter.jsx

```diff
@@ -99,30 +154,55 @@ function ModalFiles(props) {
           ? <div {...classes('empty-state')}>
             <EmptyState message={`Output contains ${numberFormatter.format(filesCount)} files and is too big to preview`} />
           </div>
-          : <ul {...classes('list')}>
-            <ListItem>
-              <input
-                {...classes('filter')}
-                value={filterText}
-                placeholder={'Filter by name or type'}
-                onChange={e => filter(e.target.value)}
-              />
-            </ListItem>
-            {fileList.map((file, key) =>
-              <FileItem
-                file={file}
-                key={key}
-                isEditable={canRemoveFiles && file.manuallyUploaded}
-                removeResult={removeResult}
-                isHidden={!filteredFilesIds[file.uuid]}
-              />
-            )}
-            {!filteredFiles.length &&
-              <ListItem mods={['borderless']}>
-                <EmptyState message={fileList.length > 0 ? 'No files that match criteria' : 'No files'} />
+          : 
+          <div>
+            <div {...classes('adress-bar')}>
+              <FileAddressBar path={filesPath} loadFolder={loadSubmissionFiles} />
+            </div>
+            <ul {...classes('list')}>
+              <ListItem {...classes('filter-box')}>
+                <input
+                  {...classes('filter')}
+                  value={filterText}
+                  placeholder={'Filter by name or type'}
+                  onChange={e => filter(e.target.value)}
+                />
               </ListItem>
-            }
-          </ul>
+              {
+                filesPath !== '/'
+                ? <FileItem
+                    file={ {
+                      docType: 'folder',
+                      label: '..',
+                      createdAt: moment(),
```

Comment:
Use datetime of some submission's file

Reply:
I'd say this field is not really needed here